package.json

```diff
@@ -84,7 +84,7 @@
     "xgettext-template": "^3.3.0"
   },
   "engines": {
-    "node": ">= 4"
+    "node": ">= 4 <8"
```

Comment:
Tested the build locally on node version 7.10.0, which is used on heroku as well.
Deployed https://open-event-frontend-app2.herokuapp.com/ Using the above setting.